### PR TITLE
New test data

### PR DESCRIPTION
**Pull request by bartstasik**

### CODE REVIEW

File: dummy_session.py [hunk 1]
```diff
@@ -113,21 +111,67 @@ def update(self, dt):
...
+        }
+        output["gazeDirection"] = {
+            "measurementId": measurementId,
+            "time": time,
+            "deviceId": deviceId + ".pupil",
+            "x": randint(0, 1088),
+            "y": randint(0, 1080),
+            "radius": 0  # To be removed in the backend
+        }
+        output["$instantaneousHeartRate"] = {
```
Comment (johnwright) [inline dummy_session.py:149]: I don't think the $ is needed here

File: dummy_session.py [hunk 2]
```diff
@@ -113,21 +111,67 @@ def update(self, dt):
...
+            "measurementId": measurementId,
+            "time": time,
+            "deviceId": deviceId + ".bodytrak",
+            "hr": randint(69, 100)
+        }
+        output["$coreBodyTemperature"] = {
```
Comment (johnwright) [inline dummy_session.py:155]: Ditto here

File: dummy_session.py (hunk 1 above)
Comment (bartstasik) [reply dummy_session.py:149]: Good spot, not sure why it didn't error out!

**Comment by charlesastaylor** [comment]
All looks great to me! The UpdateSolider mutation change I spent a while trying to figure out how to do something like that but gave up, much better!

**Pull request merged**